### PR TITLE
bug: Fix ingress update error

### PR DESCRIPTION
**Pull request by xieydd**

### CODE REVIEW

File: ingress-operator/pkg/controller/core.go [hunk 1]
```diff
@@ -253,7 +253,6 @@ func MakeAnnotations(fni *faasv1.InferenceIngress, host string) map[string]strin
 	specJSON, _ := json.Marshal(fni)
 	annotations := make(map[string]string)
 
-	annotations["kubernetes.io/ingress.class"] = class
```
Comment (kemingy) [inline ingress-operator/pkg/controller/core.go:256]: Will this affect the ingress?

Comment (xieydd) [reply ingress-operator/pkg/controller/core.go:256]: Ingress does not need this annotation when setting the ingress type.

Comment (xieydd) [reply ingress-operator/pkg/controller/core.go:256]:
```
 invalid: annotations.kubernetes.io/ingress.class: Invalid value: \"nginx\": can not be set when the class field is also set" 
```

Comment (xieydd) [reply ingress-operator/pkg/controller/core.go:256]: https://serverfault.com/questions/1061530/new-kubernetes-ingressclass-resource-and-only-specifying-ingressclassname-nginx

**Pull request merged**